install_config/revhistory_install_config.adoc

```diff
@@ -4,6 +4,31 @@
 :data-uri:
 :icons:
 :experimental:
+== Wed Apr 27 2016
+
+// tag::install_config_wed_apr_27_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Wed Apr 27 2016
+n|link:../install_config/persistent_storage/persistent_storage_nfs.html[Configuring Persistent Storage -> Persistent Storage Using NFS]
+|Updated the link:../install_config/persistent_storage/persistent_storage_nfs.html#nfs-export-settings[Export Settings] section to note the `no_wdelay` NFS export option.
+
+|link:../install_config/install/docker_registry.html[Installing -> Deploying a Docker Registry]
+|Updated the link:../install_config/install/docker_registry.html#registry-known-issues[Known Issues] section to note the `no_wdelay` NFS export option.
+
+|link:../install_config/http_proxies.html[Working with HTTP Proxies]
+|Added specific *_/etc/sysconfig_* files to the link:../admin_guide/managing_projects.html#configuring-hosts-for-proxies[Configuring Hosts for Proxies] section.
+
+|link:../install_config/http_proxies.html[Working with HTTP Proxies]
+|Added information explaining that OpenShift does not accept an asterisk as a wildcard attached to a domain suffix.
```

Comment:
These two "Working with HTTP Proxies" notes could be combined as seen here:

https://github.com/bfallonf/openshift-docs/blame/enterprise-3.1/install_config/revhistory_install_config.adoc#L40-L52